client/src/js/modules/core/components/AboutPage.tsx

```diff
@@ -2,6 +2,9 @@ import * as React from "react";
 import injectSheet from "react-jss";
 import Logo from "../../icons/components/Logo";
 import Library from "../../icons/components/Library";
+import nyuImage from "../../../../../dist/dev/images/nyu.jpeg";
```

Comment:
We want to make the images dynamic, i.e. loaded from the server. I'll show you how to do this

Reply:
Also, is the image committed to the repo? Cause I don't see anything on my end

Reply:
If you send me the links I can integrate them into the data seeding